docker-compose.yml

```diff
@@ -97,7 +97,7 @@ services:
       sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
              tango_admin --add-server SKAAlarmHandler/01 SKAAlarmhandler ska/alarmhandler/01 &&\
              sudo ln -sf /var/run/rsyslog/dev/log /dev/log &&\
-             /venv/bin/python /app/skabase/SKAAlarmHandler/SKAAlarmHandler.py 01"
+             /venv/bin/python -u /app/skabase/SKAAlarmHandler/SKAAlarmHandler.py 01"
```

Comment:
Just curious, why are you forcing unbuffered output?

Reply:
Its primarily for debugging. While testing, the docker logs wasnt showing print statements added for debugging. Adding -u was the solution I found on internet.

Reply:
Have you tried using `sys.stdout.write` rather than `print` ?

Reply:
No I did not try it. Matteo has shared me one link. I went through it and It looks like sys.stdout.write is better approach to print messages on screen for debugging purpose.

Reply:
Removed the -u option. Its purpose is served anyway.